### PR TITLE
feat(iOS,macOS) : macOS/iOS login support, and configuration editor support

### PR DESCRIPTION
**Pull request by arthur740212**
Account portal or persistent login successes in macOS/iOS
The configuration editor and config file reading both works

Latest development branch are already merged. (8/24/2022)
And the SDK is updated to the latest

Completed tasks:

macOS: 
CU-2r1rexe

iOS:
CU-rnaf0f
CU-rnacz6
CU-2tfca33

### CODE REVIEW

File: Assets/Plugins/Source/EOSManager.cs [hunk 1]
```diff
@@ -1051,7 +1051,40 @@ private void CallOnConnectLogin(Epic.OnlineServices.Connect.LoginCallbackInfo co
...
+                credentials.ExternalType = loginOptions.Credentials.Value.ExternalType;
+
+                var systemAuthCredentialsOptions = new IOSCredentialsSystemAuthCredentialsOptions();
+
+                systemAuthCredentialsOptions.PresentationContextProviding = LoginUtility_get_app_controller();
+                credentials.SystemAuthCredentialsOptions = systemAuthCredentialsOptions;
 
+                modifiedLoginOptions.Credentials = credentials;
+
+                //print(loginOptions.ScopeFlags.ToString());
```
Comment (andrew-hirata-playeveryware) [inline Assets/Plugins/Source/EOSManager.cs]: I would suggest removing these debug lines

File: Assets/Plugins/iOS/EOSManager_iOS.cs [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+        public IEOSCreateOptions CreateSystemPlatformOption()
+        {
+            var createOptions = new EOSiOSOptions();
+
+            return createOptions;
+        }
+
+        //-------------------------------------------------------------------------
+        /// <summary>
+        /// On iOS, this method handles looking up where the RTC options are.
```
Comment (andrew-hirata-playeveryware) [inline Assets/Plugins/iOS/EOSManager_iOS.cs]: This comment appears to be copied and pasted from somewhere else and should be removed.

File: Assets/Plugins/Source/EOSManager.cs (hunk 1 above)
Comment (andrew-hirata-playeveryware) [inline Assets/Plugins/Source/EOSManager.cs]: These commented out debug lines can be deleted.

**Review by andrew-hirata-playeveryware — CHANGES_REQUESTED** [review]

File: Assets/Plugins/Source/EOSManager.cs (hunk 1 above)
Comment (arthur740212) [reply Assets/Plugins/Source/EOSManager.cs]: Removed

File: Assets/Plugins/iOS/EOSManager_iOS.cs (hunk 2 above)
Comment (arthur740212) [reply Assets/Plugins/iOS/EOSManager_iOS.cs]: Removed

File: Assets/Plugins/Source/EOSManager.cs (hunk 1 above)
Comment (arthur740212) [reply Assets/Plugins/Source/EOSManager.cs]: Absolutely,  lost track of them.

**Pull request merged**